docker/docker_jit_monitor/src/main.rs

```diff
@@ -301,6 +391,12 @@ fn main() -> anyhow::Result<()> {
             // Limit our spinning if we anyway wouldn't have capacity for a new builder.
             thread::sleep(Duration::from_millis(500));
         }
+
+        thread::sleep(Duration::from_secs(5));
+        // Check if some still running images are listed as offline from github api point of view
+        if let Err(e) = kill_offline_runners(&servo_ci_scope) {
```

Comment:
I don't think we want to do this all the time, only when we detected a problem. github API calls are rate-limited, and the limit is not even too high.